template/python3/index.py

```diff
@@ -3,8 +3,15 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import sys
+import os
 from function import handler
 
+
+# distutils.util.strtobool() can throw an exception
+def is_true(val):
+    return val and val.lower() == "true" or val == "1"
```

Comment:
A complete sentence for the doc string would be better, it is clear to me because i recommended strtobool, but a future contributor won't have the context

if you want to use strtobool, you could also do this

```py
def is_true(val):
    try:
        return distutils.util.strtobool(val)
    except:
        return false
```

but either method is good to me 👍